tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -460,5 +497,169 @@ private static bool ParseBooleanConfigurationValue(string value)
                 or "y"
                 or "1";
         }
+
+        private static bool TracingWithBundle(string? profilerPathValue)
+        {
+            if (profilerPathValue is null)
+            {
+                return false;
+            }
+
+            string[] expectedEndingsForBundleSetup =
+            {
+                "/datadog/linux-musl-x64/Datadog.Trace.ClrProfiler.Native.so",
+                "/datadog/linux-x64/Datadog.Trace.ClrProfiler.Native.so",
+                "/datadog/linux-arm64/Datadog.Trace.ClrProfiler.Native.so",
+                "\\datadog\\win-x64\\Datadog.Trace.ClrProfiler.Native.dll",
+                "\\datadog\\win-x86\\Datadog.Trace.ClrProfiler.Native.dll"
+            };
+
+            foreach (var bundleSetupEnding in expectedEndingsForBundleSetup)
+            {
+                if (profilerPathValue.EndsWith(bundleSetupEnding, StringComparison.OrdinalIgnoreCase))
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        private static void NoManageCodeClrVersionIis(ProcessInfo process)
+        {
+            foreach (var module in process.Modules)
+            {
+                var fileName = Path.GetFileName(module);
+
+                if (fileName.Equals("Microsoft.AspNetCore.Server.IIS.dll", StringComparison.OrdinalIgnoreCase))
+                {
+                    AnsiConsole.WriteLine(TracingDotnetOnIis);
```

Comment:
I suggest you also check that the application pool has no managed code, and only if that check fails write to the output that this error mode was found and how to fix it

Reply:
Yeah that was the plan, but while working on it I realized we already do that [here](https://github.com/DataDog/dd-trace-dotnet/blob/ee81c2ed803ed30ee079a98da079902c0c6ec1ad/tracer/src/Datadog.Trace.Tools.Runner/CheckIisCommand.cs#L120-L123), so I ended only adding the comment above to let the TSEs or us be able to identify from the output if the app is .NET Core running on IIS  without needing to ask the customer, just in case some issues that may be facing could be related to this type of setup. WDYT? Should I move it around instead or make it more specific? Part of the other PR is making sure the outputs make sense one after the other.

Reply:
Oh I see! Sorry, I had forgotten all about that check in the IIS part. Is the other output not helpful enough for us to understand that we're running .NET Core on .NET Framework? I'd like to consolidate and reduce as much output as possible so to improve readability